src/plugins/controls/server/options_list/options_list_expensive_suggestion_queries.ts

```diff
@@ -226,4 +227,48 @@ const expensiveSuggestionAggSubtypes: { [key: string]: OptionsListSuggestionAggr
       };
     },
   },
+  /**
+   * the "date" query / parser should be used when the options list is built on a field of type date.
+   */
+  date: {
+    buildAggregation: ({ fieldName, sort, size }: OptionsListRequestBody) => {
```

Comment:
I see that you consolidated the "cheap" queries - thanks, that's much cleaner IMO!! We should be able to do the same here, too 🙇 (I know it's very annoying that we have two versions - really hoping we can figure out a way to resolve that soon....)

Reply:
Oops forgot to remove in that last commit. It's deleted now 👍